testsupport/nexus-siestajs-testsupport/src/main/resources/ft-overlay/testsuite-lib/NX/TestClass.js

```diff
@@ -153,7 +153,10 @@ Class('NX.TestClass', {
         },
         { type: 'admin[TAB]', target: this.cq1('#username') },
         { type: 'admin123', target: this.cq1('#password') },
-        { type: '[ENTER]' }
+        { type: '[ENTER]' },
+        function(next) {
+          this.waitForAnimations(next);
+        }
```

Comment:
Assuming this is the canonical source, can we now remove the duplicate copies of TestClass.js and TestHarness.js from "testsuite/functional-testsuite/src/test/ft-resources/testsuite-lib/NX" under each of the Nexus repos?